### PR TITLE
User icons

### PR DESCRIPTION
**Pull request by Snygg**

### CODE REVIEW

File: source/ShoutrApp/src/app/connected/connected.component.html [hunk 1]
```diff
@@ -15,4 +15,22 @@ <h4>Public Key</h4>
...
+</zh-peer-icon>
+<zh-peer-icon peerId="KAZOOOOOO" style="margin: 5px;">
+
+</zh-peer-icon>
+<zh-peer-icon peerId="test1234" style="margin: 5px;">
+
+</zh-peer-icon>
+<zh-peer-icon peerId="rest0123" style="margin: 5px;">
+
+</zh-peer-icon>
```
Comment (zerotwooneone) [inline source/ShoutrApp/src/app/connected/connected.component.html]: lets remove these tests
```suggestion

```

File: source/ShoutrApp/src/app/peer-icon/peer-icon.component.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.row{
+    display: flex;
+    .space{
+        height: 10px;
+        width: 10px;
```
Comment (zerotwooneone) [inline source/ShoutrApp/src/app/peer-icon/peer-icon.component.scss]: I would like to (eventually) refactor this component so the implementer decides the size of this component. 
In this case connected would set the height/width of peer-icon and the spaces would evenly distribute that space.

File: source/ShoutrApp/src/app/peer-icon/peer-icon.component.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+
+@Component({
+  selector: 'zh-peer-icon',
+  templateUrl: './peer-icon.component.html',
+  styleUrls: ['./peer-icon.component.scss']
+})
+export class PeerIconComponent implements OnChanges{
+  perSide = 5
```
Comment (zerotwooneone) [inline source/ShoutrApp/src/app/peer-icon/peer-icon.component.ts]: Consider making this an `@Input` that defaults to 5

File: source/ShoutrApp/src/app/peer-icon/peer-icon.component.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  hashCode(input:string) {
+    var hash = 0,
+      i, chr;
+    if (input.length === 0) return hash;
+    for (i = 0; i < input.length; i++) {
+      chr = input.charCodeAt(i);
+      hash = ((hash << 5) - hash) + chr;
+      hash |= 0; // Convert to 32bit integer
+    }
+    console.log(Math.abs(hash));
```
Comment (zerotwooneone) [inline source/ShoutrApp/src/app/peer-icon/peer-icon.component.ts]: remove test code
```suggestion
```

**Review by zerotwooneone — CHANGES_REQUESTED** [review]
amazing work

File: source/ShoutrApp/src/app/peer-icon/peer-icon.component.ts (hunk 3 above)
Comment (Snygg) [reply source/ShoutrApp/src/app/peer-icon/peer-icon.component.ts]: done

File: source/ShoutrApp/src/app/peer-icon/peer-icon.component.scss (hunk 2 above)
Comment (Snygg) [reply source/ShoutrApp/src/app/peer-icon/peer-icon.component.scss]: done

**Pull request merged**